syntax = "proto3";

package tesla.proto.energy_device.v1;

option go_package = "github.com/teslamotors/energy_device/pkg/protocol/protobuf/energydevice/v1";
option java_package = "com.tesla.generated.energy_device.v1";
option java_outer_classname = "AuthorizedState";

enum AuthorizedState {
  AUTHORIZED_STATE_INVALID                       = 0;
  AUTHORIZED_STATE_PENDING_VERIFICATION          = 1;
  AUTHORIZED_STATE_PENDING_VERIFICATION_TIMEOUT  = 2;
  AUTHORIZED_STATE_VERIFIED                      = 3;
  AUTHORIZED_STATE_REMOVED                       = 4;
}